pkg/frontend/admin_openshiftcluster_vmresize.go

```diff
@@ -45,6 +45,28 @@ func (f *frontend) _postAdminOpenShiftClusterVMResize(log *logrus.Entry, ctx con
 		return err
 	}
 
+	force := strings.EqualFold(r.URL.Query().Get("force"), "true")
+	if !force {
+		// Skip if node exists check on force
+		// This parameter will come in handy if the controlplane is under extreme load
+		// Acts as a workaround if API server is unavailable
+		err = f.validateMasterNodeExists(log, ctx, vmName, resourceGroupName, doc)
+		if err != nil {
+			return err
+		}
+	} else {
+		// Bland check to prevent forceful Resize of a non-master node.
+		if !strings.Contains(strings.ToLower(vmName), "master") {
```

Comment:
Per discussion in the Jira, and looking at this code, it seems sufficient to me to just rewrite the original "non forced" code to use the updated azure clients (vm level) rather than openshift (node level), if that's sufficient to determine a master node's status (which appears to be the only change here - it's a validation change, not a change to how we actually resize).

As I commented in the Jira as well, I think we can easily determine control plane status by looking at two non-openshift resources: vms and their subnets. Relying on strings like "master" is a little hacky, even though I'm sure it'll work in nearly all (if not all) cases.

Reply:
Based on the Slack discussion, we are letting this blank check live by going with the team consensus. Thanks.

Reply:
Yes correct, we are not forcing, I have corrected the comments and error messages. Thanks.